src/program/librapay.js

```diff
@@ -113,6 +89,105 @@ export async function createGenesis(
   return genesisAccount;
 }
 
+/**
+ * Load a new instance of a Move program on-chain.
+ * Returns the new program account.
+ */
+export async function publishModule(
+  connection: Connection,
+  moduleAccount: Account,
+  path: string,
+): Promise<void> {
+  try {
+    await connection.getAccountInfo(moduleAccount.publicKey);
+  } catch (e) {
+    return await MoveLoader.load(
+      connection,
+      AccountType.CompiledModule,
+      moduleAccount,
+      path,
+    );
+  }
+  console.log('Warning: module already published or account already exists');
+}
+
+/**
+ * Load a new instance of a Move program on-chain.
+ * Returns the new program account.
+ */
+export async function loadScript(
+  connection: Connection,
+  scriptAccount: Account,
+  path: string,
+): Promise<void> {
+  return await MoveLoader.load(
+    connection,
+    AccountType.CompiledScript,
+    scriptAccount,
+    path,
+  );
+}
+
+/**
+ * Mint tokens into a new Libra account.
+ * Returns the new account
+ */
+export async function runScript(
+  connection: Connection,
+  scriptAccount: PublicKey,
+  functionName: string,
+  payerAccount: Account,
+  genesisAccount: Account,
+  senderAccount: Account,
+  additionalKeys: Array<{
+    pubkey: PublicKey,
+    isSigner: boolean,
+    isWritable: boolean,
+  }>,
+  additionalSignerAccounts: Array<Account>,
+  args: ?Buffer,
+): Promise<void> {
+  const transaction = new Transaction();
+  let keys = [
+    {
+      pubkey: scriptAccount,
+      isSigner: false,
+      isWritable: false,
+    },
+    {
+      pubkey: genesisAccount.publicKey,
+      isSigner: false,
+      isWritable: false,
+    },
+    {
+      pubkey: senderAccount.publicKey,
+      isSigner: true,
+      isWritable: true,
+    },
+  ];
+  keys = keys.concat(additionalKeys);
+
+  transaction.add({
+    keys,
+    programId: MoveLoader.programId,
+    data: InstructionData.runScript(
+      senderAccount.publicKey,
+      functionName,
+      args,
+    ),
+  });
+
+  let signerAccounts = [payerAccount, senderAccount];
+  signerAccounts = signerAccounts.concat(additionalSignerAccounts);
```

Comment:
In one line `let signerAccounts = [payerAccount, senderAccount, ...additionalSignerAccounts];`

Reply:
Cool